src/diffusers/pipeline_utils.py

```diff
@@ -473,6 +491,15 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 user_agent["custom_pipeline"] = custom_pipeline
             user_agent = http_user_agent(user_agent)
 
+            if is_safetensors_available():
```

Comment:
cool that works for me - agree with your point that "calling the hub" is too hidden when fully put in `is_safetensors_compatible(...)`